api/envoy/config/core/v3/protocol.proto

```diff
@@ -157,6 +157,10 @@ message HttpProtocolOptions {
   // If this setting is not specified, the value defaults to ALLOW.
   // Note: upstream responses are not affected by this setting.
   HeadersWithUnderscoresAction headers_with_underscores_action = 5;
+
+  // Optional maximum requests for both upstream and downstream connections.
```

Comment:
Can you carry forward the verbiage ` If not specified, there is no limit. Setting this
  // parameter to 1 will effectively disable keep alive.`?

Reply:
Sure, I will do that.